dev-tools: add instructions to update flake to 0.2 (#395)
holochain/docs-pages: src/pages/get-building/upgrade-holochain.md

---
title: Holochain Upgrade 0.1 -> 0.2
hide:
  - toc
---

::: coreconcepts-intro
For existing hApps that have been written for Holochain version 0.1, these are steps to upgrade to Holochain version 0.2.
:::

### Steps to upgrade Nix flake from Holochain version 0.1 to 0.2

!!! note
The following steps only apply to Nix flakes that have been either generated by the Holochain scaffolding tool

> or follow the recommended format printed under [Dev Tools Setup](../install-advanced){target=_blank}.

Reviewer: ```suggestion
or follow the recommended format as described in [Dev Tools Setup](../install-advanced){target=_blank}.
```

!!!

> In the root folder of the hApp that should be upgraded, execute the following command in a terminal:

Reviewer: ```suggestion
In the root folder of the hApp to be upgraded, execute the following command in a terminal:
```
